rskj-core/src/test/java/co/rsk/peg/BridgeCostsTest.java

```diff
@@ -31,12 +31,10 @@
 import static org.mockito.Mockito.*;
 
 class BridgeCostsTest {
-    private static BridgeRegTestConstants bridgeConstants;
-    private static final List<BtcECKey> REGTEST_FEDERATION_PRIVATE_KEYS = Arrays.asList(
-        BtcECKey.fromPrivate(Hex.decode("45c5b07fc1a6f58892615b7c31dca6c96db58c4bbc538a6b8a22999aaa860c32")),
-        BtcECKey.fromPrivate(Hex.decode("505334c7745df2fc61486dffb900784505776a898377172ffa77384892749179")),
-        BtcECKey.fromPrivate(Hex.decode("bed0af2ce8aa8cb2bc3f9416c9d518fdee15d1ff15b8ded28376fcb23db6db69"))
+    private static final List<BtcECKey> fedECKeys = BitcoinTestUtils.getBtcEcKeysFromSeeds(
```

Comment:
```suggestion
    private static final List<BtcECKey> fedBtcECKeys = BitcoinTestUtils.getBtcEcKeysFromSeeds(
```
careful, ECKeys is for rsk and mst keys.

Reply:
how's that?

Reply:
Just in case, it is fixed already!